README.md

```diff
@@ -321,6 +321,36 @@ and add them to the media library if their MIME types are supported. If the
 file/folder in _path_ does not exist/is not readable or is not provided then an
 error will be returned and the corresponding log message would be written into logs.
 
+## Internal Audio & Video Recording
+
+Required steps to activate recording:
+
+```bash
+adb shell pm grant io.appium.settings android.permission.RECORD_AUDIO
+adb shell appops set io.appium.settings PROJECT_MEDIA allow
+```
+
+Start Recording:
+```bash
+adb shell am start -n "io.appium.settings/io.appium.settings.Settings" -a io.appium.settings.recording.ACTION_START --es filename abc.mp4 --es priority 2 --es max_duration_sec 900 --es resolution_mode 5
+```
+
+### Arguments (see above start command as an example for giving arguments)
+- filename (Mandatory) - You can type recording video file name as you want, but recording currently supports only "mp4" format so your filename must end with ".mp4"
+- priority (Optional) - Default value: 2 which means recording thread priority is Maximum however if you face performance drops during testing with recording enabled, you can reduce recording priority to 1 - Normal or 0 - Minimum
+- max_duration_sec (Optional) (in seconds) - Default value: 900 seconds which means maximum allowed duration is 15 minute, you can increase it if your test takes longer than that
+- resolution_mode (Optional) - Default value: 5 which means default resolution for recording is Full HD 1920x1080 however you can change it to following resolutions: 4 - HD (1280x720), 3 - 480P (720x480), 2 - QVGA (320x240), 1 - QCIF (176x144)
+
```

Comment:
Ideas for the future:
It would be nice to have FPS value configurable

Reply:
+1 yeah, noted, will try to sum up and write as comment all future-directions after the PR 👍